.gitignore

```
@@ -13,7 +13,7 @@
 
 # misc
 .DS_Store
-.env.local
+# .env.local
```

pleas uncomment this line so that the env file can be ignored

```suggestion
.env.local
```